WSSiOS/WSSiOS/Source/Presentation/NovelDetail/NovelDetailView/NovelDetailHeaderView/NovelDetailHeaderAssistantView/NovelDetailHeaderReadInfoButton.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  NovelDetailHeaderReadInfoButton.swift
+//  WSSiOS
+//
+//  Created by YunhakLee on 8/20/24.
+//
+
+import UIKit
+
+import SnapKit
+import Then
+
+final class NovelDetailHeaderReadInfoButton: UIButton {
+    
+    //MARK: - Properties
+    
+    let iconImage: UIImage
+    
+    //MARK: - Components
+    
+    private let stackView = UIStackView()
+    private let iconImageView = UIImageView()
+    private let readInfoLabel = UILabel()
+    private let navigationImageView = UIImageView()
+    
+    //MARK: - Life Cycle
+    
+    init(iconImage: UIImage) {
+        self.iconImage = iconImage
+        
+        super.init(frame: .zero)
+        
+        setUI()
+        setHierarchy()
+        setLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    //MARK: - UI
+    
+    private func setUI() {
+        self.do {
+            $0.backgroundColor = .wssWhite
+            $0.layer.cornerRadius = 11.5
```

Comment:
```suggestion
            $0.layer.cornerRadius = 14
```

W1;
14라고 나와있네요! 근데 두 값 다 라운드가 같은 느낌?!?!.... 사이즈가 작아서 그런가 ㅎㅎ 그래도 피그마상엔 14로 나와있으니 수정하는 게 좋을 것 같아용

Reply:
height / 2 라서 11.5 맞는 거 같아요!

Reply:
height 프로퍼티를 추가해서 좀 더 직관적으로 변경했어요 !